WNPRC_EHR/src/org/labkey/wnprc_ehr/notification/NotificationToolkit.java

```diff
@@ -956,15 +956,23 @@ public String createAnimalHistoryURL(Container c, String subject) {
     public Boolean checkIfAnimalIsAlive(Container c, User u, String idToCheck) {
         try {
             //Runs query.
-            ArrayList<String> animalDemographicRow = getTableRowAsList(c, u, "study", "Demographics", null, "id", idToCheck, new String[]{"calculated_status"});
+//            ArrayList<String> animalDemographicRow = getTableRowAsList(c, u, "study", "Demographics", null, "id", idToCheck, new String[]{"calculated_status"});
 
-            //Checks alive status.
-            if (animalDemographicRow.get(0).equals("Alive")) {
-                return true;
-            }
-            else {
-                return false;
+            // Creates filter.
+            SimpleFilter myFilter = new SimpleFilter("id", idToCheck, CompareType.EQUAL);
+            // Creates columns to retrieve.
+            String[] targetColumns = new String[]{"calculated_status"};
+            //Runs query.
+            ArrayList<HashMap<String,String>> returnArray = getTableMultiRowMultiColumnWithFieldKeys(c, u, "study", "Demographics", myFilter, null, targetColumns);
```

Comment:
Make this a List instead of a ArrayList

Reply:
I'll do this in the next PR if that's ok.  There's a lot of ArrayList uses I'll need to go in and change through multiple files.